draft-timbru-sidrops-publication-server-bcp-00.md

```diff
@@ -202,35 +203,34 @@ Whenever there is an update:
   - fix the timestamps of files (see next section)
   - change the symlink to point to the new directory
 
+An example implementation of the above steps is available as a POSIX
+shellscript [@rsync-move].
+
 This way, rsyncd does not need to restart, and since rsyncd resolves this
 symlink when it `chdir`s into the module directory when a client connects, any
 connected RPs will be able to read a consistent state.
 
 The repository can remove old directories when no RP fetching at a reasonable
 rate is reading that data. It's hard to determine this in practice. Empirical
 data suggests that Rsync Repositories MAY assume that it is safe to do so after
-one hour. We recommend repository operators monitor for "file has vanished"
-lines in the rsync log file to detect how many clients are affected by these
-deletions.
+one hour or two hours. We recommend repository operators monitor for "file has
```

Comment:
Tom Harrisson said that in their experience one hour is enough.

I don't have a big issue with saying two hours to be even more sure, but I don't like saying "one or two" here. I think we should make a choice.